ports/geckolib/gecko_style_structs.rs

```diff
@@ -0,0 +1,891 @@
+/* automatically generated by rust-bindgen */
```

Comment:
This is awesome! Way better than the 1.4 megabytes of generated code that I came up with. ;-)